dbt_common/__about__.py

```diff
@@ -1 +1 @@
-version = "1.7.0"
+version = "1.8.0a1"
```

Comment:
If I don't do this, `hatch` will not install the version on GitHub since the version matches what's in PyPI. This will become 1.8.0 at some point so this feels right.